### PR TITLE
make signing key name an optional flag

### PR DESCRIPTION
**Pull request by rawlingsj**
Signed-off-by: James Rawlings <jrawlings@chainguard.dev>

### CODE REVIEW

File: pkg/commands/pod.go [hunk 1]
```diff
@@ -195,7 +195,7 @@ rm ${KEY}
 touch start-gsutil-cp
 echo exiting $success...
 exit $success
-`, arch, strings.Join(targets, " ")),
+`, signingKeyName, signingKeyName, signingKeyName, signingKeyName, arch, strings.Join(targets, " ")),
```
Comment (imjasonh) [inline pkg/commands/pod.go:198]: `//TODO(jason): Use templating :)`

File: pkg/commands/pod.go [hunk 2]
```diff
@@ -180,11 +180,11 @@ if [[ ! -f /var/secrets/melange.rsa ]]; then
...
-  cp /var/secrets/melange.rsa wolfi-signing.rsa
-  wget -O wolfi-signing.rsa.pub https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
-  KEY=wolfi-signing.rsa
-  cat wolfi-signing.rsa.pub
-  ls wolfi-signing.rsa
+  cp /var/secrets/melange.rsa %s.rsa
+  cp packages/*.rsa.pub .
+  KEY=%s.rsa
+  cat %s.rsa.pub
+  ls %s.rsa
```
Comment (imjasonh) [inline pkg/commands/pod.go:187]: This isn't really necessary, now that we `cat` it. I'm not sure why it was ever there. Can we just remove it?

Comment (rawlingsj) [reply pkg/commands/pod.go:187]: yeah lemme do that now

File: pkg/commands/pod.go (hunk 1 above)
Comment (rawlingsj) [reply pkg/commands/pod.go]: Yeah or I'm happy to do that in a followup

**Review by imjasonh — APPROVED** [review]

**Pull request merged**